src/snowflake/snowpark/functions.py

```diff
@@ -1371,6 +1386,29 @@ def upper(e: ColumnOrName) -> Column:
     c = _to_col_if_str(e, "upper")
     return builtin("upper")(c)
 
+def struct(*cols)    -> Column:
+    """Creates a new object column.
+    Example::
+        >>> df = session.create_dataframe(['a', 'b', 'c'], schema=["a", "b", "c"])
+        >>> df.select(struct("a", "b", "c")).collect()
+    """
```

Comment:
```suggestion
def struct(*cols) -> Column:
    """Creates a new object column.
    
    Example::
        >>> df = session.create_dataframe(['a', 'b', 'c'], schema=["a", "b", "c"])
        >>> df.select(struct("a", "b", "c")).collect()
    """
```

please also add output that come from `.collect()`

Reply:
output added